benchmarks/benches/crypto_bench.rs

```diff
@@ -301,6 +398,6 @@ fn criterion() -> Criterion {
 criterion_group! {
     name = benches;
     config = criterion();
-    targets = keys_query, keys_claiming, room_key_sharing, devices_missing_sessions_collecting,
+    targets = load_members_benchmark, keys_query, keys_claiming, room_key_sharing, devices_missing_sessions_collecting,
```

Comment:
As noted, this benchmark should go into a different file. Could be its own file, maybe `room.rs` or something like that.